### PR TITLE
Update README.md with Core constructor parameters

### PR DESCRIPTION
**Pull request by louilinn**
Fix comma in code block

### CODE REVIEW

**Comment by juanenrisley** [comment]
> Maybe we can also specify if there are other option

The whole idea is to deprecate the cli type and have only the server type, but because of retrocompatibility we'll have both types for the time being.

File: README.md [hunk 1]
```diff
@@ -91,10 +91,10 @@ const core = new CirclesCore(web3, {
   proxyFactoryAddress: '0x9b1f7F645351AF3631a656421eD2e40f2802E6c0',
   safeMasterAddress: '0x59d3631c86BbE35EF041872d502F218A39FBa150',
   apiServiceEndpoint: 'http://api.circles.local',
-  pathfinderServiceEndpoint: 'http://pathfinder.circles.local'
-  pathfinderType: 'server',
+  pathfinderServiceEndpoint: 'http://pathfinder.circles.local',
+  pathfinderType: 'server', // other option: 'cli'
   graphNodeEndpoint: 'http://graph.circles.local',
-  databaseSource: 'graph',
+  databaseSource: 'graph', // other options:
```
Comment (llunaCreixent) [inline README.md:97]: No other options available yet. Adding the comment for `databaseSource` seems confusing

Comment (louilinn) [reply README.md:97]: ok but maybe revert the last commit and still fix the comma, since the code formatting in readme is wrong

Comment (louilinn) [reply README.md:97]: I am just lost on how to make the latest core version work with the old api

Comment (llunaCreixent) [reply README.md:97]: Ok, then setting the pathfinder type to "cli" is enough

**Review by llunaCreixent — APPROVED** [review]

**Pull request merged**